codegen/lib/graphql_java_gen/templates/APISchema.java.erb

```diff
@@ -236,7 +236,7 @@ public class <%= schema_name %> {
 
                 <% fields.each do |field| %>
                     <%= java_doc(field) %>
-                    <%= java_annotations(field) %>
+                    <%= java_annotations(field) -%>
```

Comment:
do not generate new line in case if `java_annotations(field)` resolved to blank line